test/controller.test.ts

```diff
@@ -530,6 +536,645 @@ contract('Controller', ([owner, accountOwner1, accountOperator1, random]) => {
           'Controller: otoken used as collateral is already expired',
         )
       })
+
+      it('should revert when vault have more than 1 long otoken', async () => {
+        const expiryTime = new BigNumber(60 * 60) // after 1 hour
+        const collateralToDeposit = new BigNumber('20')
```

Comment:
```suggestion
        const longToDeposit = new BigNumber('20')
```

Reply:
This is collateral asset not long otoken

Reply:
hmm but you are calling the DepositLong operate action?